airflow/gcp/example_dags/example_speech.py

```diff
@@ -29,9 +29,9 @@
 
 from airflow.utils import dates
 from airflow import models
-from airflow.contrib.operators.gcp_text_to_speech_operator import GcpTextToSpeechSynthesizeOperator
-from airflow.contrib.operators.gcp_speech_to_text_operator import GcpSpeechToTextRecognizeSpeechOperator
-from airflow.contrib.operators.gcp_translate_speech_operator import GcpTranslateSpeechOperator
+from airflow.gcp.operators.text_to_speech import GcpTextToSpeechSynthesizeOperator
+from airflow.gcp.operators.speech_to_text import GcpSpeechToTextRecognizeSpeechOperator
```

Comment:
Maybe we should probably have a single file for `speech_to_text` & `text_to_speech` ???

Reply:
There are two product / APIs:
- Cloud Text-to-Speech API
- Cloud Speech-to-Text API

We could put those under "conversation" like here:
https://cloud.google.com/products/ai/?tab=tab3
but I am not sure if user will know where to look for the operators.

Reply:
👍 Lets keep it as it is in that case